dietpi/dietpi-software

```diff
@@ -8591,7 +8590,7 @@ _EOF_
 					if G_WHIP_INPUTBOX "${invalid_text}Please enter the obfs4 port that people will use to connect to your bridge:
 \nNB: This port needs to be forwarded by your router and/or opened in your firewall settings. Default value is: 80. Do not use default value if running a webserver.
 \nDo not use 9001, as this port is commonly associated with Tor and censors might scan for it." && disable_error=1 G_CHECK_VALIDINT "$G_WHIP_RETURNED_VALUE" 1; then
-						orport=$G_WHIP_RETURNED_VALUE
+						obfs4port=$G_WHIP_RETURNED_VALUE
```

Comment:
It needs to be adjusted below in the /etc/tor/torrc as well, where it's currently: `ORPort $orport`

Reply:
This is specifically the `obfs4` port for the bridge.  It uses both.